pandas/core/config_init.py

```diff
@@ -287,6 +285,23 @@ def table_schema_cb(key):
     _enable_data_resource_formatter(cf.get_option(key))
 
 
+def is_terminal():
+    """
+    Detect if Python is running in a terminal.
+
+    Returns True if Python is running in a terminal or False if not.
+    """
+    try:
+        ip = get_ipython()
```

Comment:
I would expect flake8 to complain about this (F821 I think).  Any idea why it doesn’t?

Low priority: could make the docstring follow the standard format

Reply:
> I would expect flake8 to complain about this (F821 I think)

Uncertain.  I don't think it's because of the `try-except` block wrapping it?

> Low priority: could make the docstring follow the standard format

Agreed on priority.  Can always do after.